config/routes.rb

```diff
@@ -24,6 +24,14 @@
   get "/connect", to: redirect("https://openfoodnetwork.org/#{ENV['DEFAULT_COUNTRY_CODE'].andand.downcase}/connect/")
   get "/learn", to: redirect("https://openfoodnetwork.org/#{ENV['DEFAULT_COUNTRY_CODE'].andand.downcase}/learn/")
 
+  get "/cart", :to => "spree/orders#edit", :as => :cart
```

Comment:
Did you copy these just for clarity that we are using Spree code here? I think that's a good idea.

Reply:
yes! all related routes on ofn's side now.